Scripts/Spec/GLTFMesh.cs

```diff
@@ -267,6 +295,10 @@ private class BlendShape {
 						continue;
 					}
 
+					RenderingMode mode = RenderingMode.TRIANGLES;
+					for (int k = 0; k < meshes[i].primitives.Count; k++)
+						mode = meshes[i].primitives[k].mode;
+
```

Comment:
I understand that while GLTF supports different rendering modes per primitive, Unity only supports it per-mesh. This means you have to pick just one primitive mode and use that on the entire mesh. But why are you looping through all primitives to get it? Why not just get the first with
`mode = meshes[i].primitives[0].mode`?

Additionally, it doesn't seem like you use this mode value anywhere. Remove it?

Reply:
Yep, this was a left-over from a previous implementation. I will remove it. Unity supports the mode at the submesh-level.